### PR TITLE
docs(relnote): Update Firefox release for "offset-position"

### PR DESCRIPTION
**Pull request by dipikabh**
### Description

As per my comment [here](https://github.com/mdn/content/issues/27174#issuecomment-1631717361), I am updating the release version from 115 to 116.

The `normal` keyword is available in Fx116 beta (0b6) behind the preference `layout.css.motion-path-offset-position.enabled`. For the code in [codepen](https://codepen.io/dipikabh/pen/oNQZyOm) to work as expected, the preference `layout.css.motion-path-ray.enabled` also needs to be enabled.

- Previous PR where this property was added to experimental_features: https://github.com/mdn/content/pull/27730
- Doc issue tracker: https://github.com/mdn/content/issues/27174
- Related BCD update: https://github.com/mdn/browser-compat-data/pull/20250
- Firefox bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1559232

### CODE REVIEW

**Review by AustinTrang1995 — COMMENTED** [review]
ok

**Comment by bsmth** [comment]
Thanks for the ping - taking a look 👀

**Review by bsmth — APPROVED** [review]
Thanks @dipikabh 👍🏻

File: files/en-us/mozilla/firefox/experimental_features/index.md [hunk 1]
```diff
@@ -821,22 +821,22 @@ The CSS {{cssxref("offset-position")}} property defines the initial position of
   <tbody>
     <tr>
       <th>Nightly</th>
-      <td>115</td>
-      <td>Yes</td>
+      <td>116</td>
+      <td>No</td>
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/experimental_features/index.md:825]: Actually it looks like it's [enabled in nightly](https://hg.mozilla.org/integration/autoland/file/tip/modules/libpref/init/StaticPrefList.yaml#l8619)

```suggestion
      <td>Yes</td>
```

**Pull request merged**